commit 10bd563423a05225ed9ee16f4631c157fd6104c5 (refs/changes/59/2359/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-08-09T12:24:40-07:00 (2 years, 2 months ago)
    
    joyent/node-cueball#119 bump min version of mooremachine to get timestamps

diff --git a/CHANGES.adoc b/CHANGES.adoc
index f7627c9..d436f2f 100644
--- a/CHANGES.adoc
+++ b/CHANGES.adoc
@@ -6,6 +6,18 @@ toc::[]
 
 ## v2.x
 
+### v2.2.9
+
+Maintenance release.
+
+Bugs fixed:
+
+ - #118 slot should retain reference to previous claim handle (helps in
+   debugging some types of claimer misbehaviour)
+ - #112 docs improvements
+ - #119 bump min version of mooremachine to get timestamps (another debugging
+   aid, to see how long claims took to process)
+
 ### v2.2.8
 
 Maintenance release.
diff --git a/package.json b/package.json
index ee0b65c..ced0d14 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "2.2.8",
+  "version": "2.2.9",
   "description": "manage a pool of connections to a multi-node service where nodes are listed in DNS",
   "main": "lib/index.js",
   "dependencies": {
@@ -9,7 +9,7 @@
     "cmdutil": ">=1.0.0 <2.0.0",
     "extsprintf": ">=1.3.0 <2.0.0",
     "ipaddr.js": ">=1.1.0 <2.0.0",
-    "mooremachine": ">=2.1.0 <3.0.0",
+    "mooremachine": ">=2.2.0 <3.0.0",
     "mname-client": ">=0.5.0 <0.6.0",
     "uuid": ">=3.0.1 <4.0.0",
     "posix-getopt": ">=1.2.0 <2.0.0",
